packages/dx-grid-core/src/plugins/local-grouping/computeds.test.js

```diff
@@ -31,9 +31,10 @@ describe('LocalGrouping computeds', () => {
     });
   });
 
-  const groupRow = ({ groupedBy, ...restParams }) => ({
+  const groupRow = ({ groupedBy, collapsedItems = [], ...restParams }) => ({
```

Comment:
Is it ok that even expanded group row has collapsed items?

Reply:
Why do we call it `items` here? Seems like it should be called `rows`. It is important as it may be a part of the public interface.